content/en/user-guide/aws/eks/index.md

```diff
@@ -551,6 +549,27 @@ spec:
   restartPolicy: Always
 ```
 
+## Supported Versions
+
+LocalStack uses [k3s](https://github.com/k3s-io/k3s) under the hood for creating EKS clusters.
+Below is the list of supported Kubernetes versions and their corresponding k3s versions.
+The default version is `1.22`.
+
+| Kubernetes Version | k3s Version       |
+|---------------------|-------------------|
+| 1.30               | v1.30.4-k3s1     |
+| 1.29               | v1.29.0-k3s1     |
+| 1.28               | v1.28.5-k3s1     |
+| 1.27               | v1.26.9-k3s1     |
+| 1.26               | v1.26.12-k3s1    |
+| 1.25               | v1.25.9-k3s1     |
+| 1.24               | v1.24.13-k3s1    |
+| 1.23               | v1.23.17-k3s1    |
+| 1.22 (Default)     | v1.22.6-k3s1     |
```

Comment:
These versions recently changed, and 1.31 + 1.32 is now available. I will send you the link!